src/tests/ftest/pool/simple_create_delete_test.py

```diff
@@ -76,13 +77,19 @@ def test_create(self):
                 break
 
         try:
-            self.pool = DaosPool(self.context)
-            self.pool.create(mode, uid, gid, 1073741824, setid)
-            if expected_result in ['FAIL']:
+            self.pool = TestPool(
+                self.context, dmg_command=self.get_dmg_command())
+            self.pool.mode.update(mode)
```

Comment:
This does not en up using `dmg` for the `control_method` without the `self.pool.get_params()` call and the `control_method: dmg` yaml file entry